scalableminds/webknossos
# Fix "Download Meshes" functionality

Avoid that zip.js dynamically creates a web worker using new Blob(...) which violates the CSP.
The z-worker.js seems to be requested for each mesh :shrug: but it's cached so it shouldn't be an issue.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Load a couple of meshes
- Use "Download Meshes" in the context menu of a group in the segments tab (this fails on master due to CSP)
- Check that the single mesh download still works

------
(Please delete unneeded items, merge only when none are left open)
